src/rain-report/rain-report.controller.ts

```diff
@@ -9,4 +17,10 @@ export class RainReportController {
   async index() {
     return await this.rainReportService.all();
   }
+
+  @Post()
+  @HttpCode(HttpStatus.CREATED)
```

Comment:
NestJS is smart enough to know when a resource has been created and include HTTP status `201 CREATED` in the response, but we explicitly document the expectation here as well.